### PR TITLE
Simplify google connect process

### PR DESCRIPTION
**Pull request by snake14**
### Description:

This is implementing the Google OAuth UI redesign to streamline the process for customers.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by AltamashShaikh — COMMENTED** [review]
Just testing the onpremise flow, overall looks good, but see few issues
When we upload a json, we should change the text to uploading of button and disable the button click

We used to a show a notification at top which we have removed, but that notification has a important info that each import will create a new site, this is essential for CLOUD since there we have limit on no of sites that can be created

**Review by AltamashShaikh — COMMENTED** [review]
@snake14 Overall the testing looks good for both on-premise and cloud

**Comment by snake14** [comment]
> Just testing the onpremise flow, overall looks good, but see few issues When we upload a json, we should change the text to uploading of button and disable the button click
> 
> We used to a show a notification at top which we have removed, but that notification has a important info that each import will create a new site, this is essential for CLOUD since there we have limit on no of sites that can be created

Thank you @AltamashShaikh . That's a good idea about the upload button. I'll work on implementing that. I tagged PO on the ticket about the notification removal.

**Comment by snake14** [comment]
@AltamashShaikh Stan pointed out that the first paragraph in the new design ends with `Matomo will automatically create a new website for each one.`, so we're alright removing the notice.

**Comment by AltamashShaikh** [comment]
> @AltamashShaikh Stan pointed out that the first paragraph in the new design ends with `Matomo will automatically create a new website for each one.`, so we're alright removing the notice.

Cool :+1:

File: Controller.php [hunk 1]
```diff
@@ -163,6 +186,17 @@ public function index($errorMessage = false)
                     ],
                 ],
             ],
+            'isConnectAccountsActivated' => $isConnectAccountsActivated,
+            'radioOptions' => [
+                'connectAccounts' => 'Quick connect with Google Analytics (recommended)',
```
Comment (AltamashShaikh) [inline Controller.php]: @snake14 This should be in a translation file

File: Controller.php [hunk 2]
```diff
@@ -163,6 +186,17 @@ public function index($errorMessage = false)
...
+                'manual' => 'Advanced Google OAuth client configuration',
```
Comment (AltamashShaikh) [inline Controller.php]: same here

**Review by AltamashShaikh — APPROVED** [review]
@snake14 I tested this by moving the `ConnectAccounts` plugin folder to `tmp` ` mv ../ConnectAccounts /tmp/` to test if `use` statement will fail, but it works as expected :smile: 

Left 2 comments for translation, rest looks good :+1:

File: Controller.php (hunk 1 above)
Comment (snake14) [reply Controller.php]: Thank you @AltamashShaikh . I'll move those strings to a translation file.

**Pull request merged**